app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -62,15 +64,20 @@ export default {
       }
       this.findIdOfParam()
 
-      application.router.navigate(`league/${this.clanIdSelected}` , { trigger: true })
+      const leagueURL = this.clanIdSelected ? `league/${this.clanIdSelected}` : 'league'
+      application.router.navigate(leagueURL)
+      // { trigger: true } does not work here due to Vue routing weirdness, so we change URL and reload:
+      location.reload()
     },
 
     findIdOfParam () {
       if (this.clanIdOrSlug) {
         this.clanIdSelected = (this.clanByIdOrSlug(this.clanIdOrSlug) || {})._id
         this.loadClanRequiredData({ leagueId: this.clanIdSelected })
+        this.loadCodePointsRequiredData({ leagueId: this.clanIdSelected })
       } else {
         this.loadGlobalRequiredData()
```

Comment:
There may be an issue here causing an error in the console but everything runs ok - will know more soon.